arelle/RuntimeOptions.py

```diff
@@ -0,0 +1,149 @@
+from arelle.typing import TypeGetText
+from arelle.SystemInfo import hasWebServer
+from dataclasses import InitVar, dataclass
+from typing import Optional, Union
+
+_: TypeGetText
+
+RuntimeOptionValue = Union[bool, int, str, None]
+
+
+class RuntimeOptionsException(Exception):
+    pass
+
+
+@dataclass()
+class RuntimeOptions:
+    pluginOptions: InitVar[Optional[dict[str, RuntimeOptionValue]]] = None,
+    abortOnMajorError: Optional[bool] = None,
+    about: Optional[str] = None,
+    anchFile: Optional[str] = None,
```

Comment:
You need to removing the trailing commas from these fields. With them Python treats this as one very large comma separated tuple.
```suggestion
    anchFile: Optional[str] = None
```